### DISCUSSION TITLE
File explorer delete feature

### DISCUSSION THREAD

**Discussion by x0rld** [undergroundwires/privacy.sexy#278] [Q&A] [answered]
How windows is suppose to work without file explorer ?
The progam itself is supposed to hold all the window managment beside beeing a file explorer ?

**Comment by undergroundwires** [comment, marked as answer]
Hi, this is not the default File Explorer (`explorer.exe`) but something else.

After your question, I did a bit research.

It looks like this:
![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/722c952a-ec5b-4c29-ae4e-64a85fbac516)

You can test it by running the following command in command prompt:

 ```batchfile
 explorer shell:AppsFolder\c5e2524a-ea46-4f67-841f-6a9465d9d515_cw5n1h2txyewy!App
 ````

If you delete this app, the normal / default file manager works without any issues, and this command starts (falls back to)`explorer.exe`. Seems to be an unused feature. We should add more documentation to this.
